src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/workbookheader/WorkbookHeader.kt

```diff
@@ -0,0 +1,83 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.workbookheader
+
+import com.jfoenix.controls.JFXCheckBox
+import javafx.beans.property.SimpleDoubleProperty
+import javafx.beans.property.SimpleObjectProperty
+import javafx.beans.property.SimpleStringProperty
+import javafx.scene.layout.Priority
+import javafx.scene.layout.VBox
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.jvm.statusindicator.control.statusindicator
+import tornadofx.*
+import java.util.function.Predicate
+
+data class FilterOption<T>(val text: String, val predicate: Predicate<T>)
```

Comment:
It feels odd to me to store the filtering rules (predicate) in the header widget.

Reply:
Agreed, the widget should allow for:
 - customizing the filter checkbox text
 - expose a property to observe the filter state (on or off)
 - allow for changing filter state

Views are basically never the place to house real work being done